aries-site/src/data/wcag/components.json

```diff
@@ -47,6 +47,405 @@
+        "status": "passed",
+        "notes": "Uses background color and border to indicate state."
+      },
+      {
+        "rule": "1.4.2",
+        "label": "Audio Control",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.3",
+        "label": "Contrast (Minimum)",
+        "status": "passed",
+        "notes": "Minimum contrast ratio of 6.29 (selected state) exceeds 4.5:1."
+      },
+      {
+        "rule": "1.4.4",
+        "label": "Resize Text",
+        "status": "passed"
+      },
+      {
+        "rule": "1.4.5",
+        "label": "Images of Text",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.6",
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": "Minimum contrast ratio of 6.29 (selected state) does not exceed 7:1."
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": "Minimum contrast ratio of 6.29 exceeds 3:1."
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "passed",
+        "notes": "Hover and focus states are indicated by background color and border."
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "passed"
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "passed"
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "passed"
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "passed"
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "passed"
```

Comment:
This rule is passing, but I feel like we could group it under 'not-applicable' for components that don't have any animation elements.

Reply:
I evaluated this based on strict reading of the criteria which reads:

"Web pages **do not contain** anything that flashes more than three times in any one second period, or the flash is below the general flash and red flash thresholds."

but am open to marking as "not-applicable" to reduce clutter/noise.

Reply:
done